tests/conftest.py

```diff
@@ -1,15 +1,20 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import os
 import re
+import sys
 from datetime import timedelta
 
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.dirname(__file__))))
```

Comment:
I still don't see a good reason to change `sys.path`

You can easily run py.tests using an IDE or just running the command:
`~/workspace/git/knowit> py.test tests/test_mediainfo.py`